network.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+module.exports = function (seneca) {
+  seneca.listen()
+    .client({type: 'web', port: 10305, pin: {role: 'cd-badges', cmd: '*'}})
```

Comment:
I realise you just cut and pasted this, but it really bugs me that value for pin is an object here, and a string for the others...